Hudl.Mjolnir.SystemTests/packages.config

```diff
@@ -4,5 +4,10 @@
   <package id="Hudl.Config" version="1.6.0" targetFramework="net45" />
   <package id="log4net" version="2.0.0" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="6.0.2" targetFramework="net45" />
-  <package id="xunit" version="1.9.2" targetFramework="net45" />
+  <package id="xunit" version="2.0.0" targetFramework="net45" />
+  <package id="xunit.abstractions" version="2.0.0" targetFramework="net45" />
+  <package id="xunit.assert" version="2.0.0" targetFramework="net45" />
+  <package id="xunit.core" version="2.0.0" targetFramework="net45" />
+  <package id="xunit.extensibility.core" version="2.0.0" targetFramework="net45" />
+  <package id="xunit.runner.visualstudio" version="2.0.0" targetFramework="net45" />
```

Comment:
The SystemTests don't need it (yet?) - they're more standalone and long-running and aren't kicked off by TeamCity. I've only run them manually from VisualStudio.